spec/models/grade_entry_form_spec.rb

```diff
@@ -0,0 +1,241 @@
+require 'spec_helper'
+require File.expand_path(File.join(File.dirname(__FILE__), '..', '..', 'test', 'test_helper'))
```

Comment:
You can delete these two lines. And code that was necessary from the old test_helper should be moved into a new helper under `spec/`.